packages/tldraw/src/lib/tools/SelectTool/childStates/PointingHandle.ts

```diff
@@ -37,10 +37,19 @@ export class PointingHandle extends StateNode {
 
 	override onPointerMove: TLEventHandlers['onPointerMove'] = () => {
 		if (this.editor.inputs.isDragging) {
-			this.parent.transition('dragging_handle', this.info)
+			this.kickoff()
 		}
 	}
 
+	override onLongPress: TLEventHandlers['onLongPress'] = () => {
+		this.kickoff()
+	}
+
+	private kickoff() {
```

Comment:
let's rename `kickoff` - it's hard to know what it's kicking off :P 
for example, we might be kicking off a drag or a translate or different things depending on the current inputs in this state tool.

(here and elsewhere in this PR)

Reply:
Renamed to `startTranslating`, `startRotating`, etc.